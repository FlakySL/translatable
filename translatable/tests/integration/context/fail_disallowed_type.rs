#[allow(unused_imports)] // trybuild
use ::{std::collections::HashMap, translatable::translation_context};

#[translation_context(base_path = greetings)]
struct Context {
    formal: i32,
    informal: String,
}

#[allow(unused)]
fn main() {} // trybuild

