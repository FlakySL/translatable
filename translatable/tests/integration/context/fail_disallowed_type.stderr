error: Only String' and '&str' is allowed for translation contexts
 --> tests/integration/context/fail_disallowed_type.rs:4:1
  |
4 | #[translation_context(base_path = greetings)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `translation_context` (in Nightly builds, run with -Z macro-backtrace for more info)
